core/templates/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -183,7 +188,16 @@ angular.module('oppia').directive('ckEditor4Rte', [
               items: ['Source']
             }
           ]
-        });
+        };
+
+        if (scope.uiConfig()) {
+          ckConfig = _updateCKEditorConfig(ckConfig, scope.uiConfig());
```

Comment:
Instead of having some config related keys/value in line 129 and a few in a separate function and passing dicts and scope as aparam, can we just have _getCKEditorConfig(<without any param>)? [not sure!]

Reply:
The config here is fairly intertwined with other logic in the `link` function. I tried separating it out and I think it's more readable this way, but I'd like a second opinion to make sure.